test/integration/component/test_blocker_bugs.py

```diff
@@ -663,7 +663,14 @@ def test_01_restart_network_cleanup(self):
         #    all it's services should resume
 
         # Find router associated with user account
-        list_router_response = list_routers(
```

Comment:
We already have the list_routers as wrapper function on top of the api building. This function serves basic and advanced zone depending upon the passed arguments. Hence the same is called depending on the network type check. Adding another function with network type as a parameter might just be another wrapper that internally calls list_routers again.